djtriggers/management/commands/clean_triggers.py

```diff
@@ -42,7 +42,9 @@ def handle_noargs(self, **options):
         # Delete each trigger
         for trigger in to_be_deleted:
             # Delete the specific trigger information
-            table = settings.DJTRIGGERS_TYPE_TO_TABLE[trigger.trigger_type]
+            table = getattr(settings.DJTRIGGERS_TYPE_TO_TABLE, trigger.trigger_type, None)
+            if table is None:
+                continue
```

Comment:
Trigger will not be deleted, is this the wanted behavior? see https://github.com/vikingco/django-triggers/pull/8/files#diff-e1346951892cf8f0978cb1279778964cR55